pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -234,6 +234,12 @@ func MakeCephExporterMetricsService(cephCluster cephv1.CephCluster, servicePortM
 func EnableCephExporterServiceMonitor(context *clusterd.Context, cephCluster cephv1.CephCluster, scheme *runtime.Scheme, opManagerContext context.Context, servicePortMetricName string) error {
 	serviceMonitor := k8sutil.GetServiceMonitor(cephExporterAppName, cephCluster.Namespace, servicePortMetricName)
 
+	if cephCluster.Spec.Monitoring.Interval != nil {
+		duration := cephCluster.Spec.Monitoring.Interval.Duration.String()
+		serviceMonitor.Spec.Endpoints[0].Interval = monitoringv1.Duration(duration)
+		logger.Infof("redo: updating exporter interval with new value %s", duration)
```

Comment:
Make it debug, or perhaps just remove it?
```suggestion
		logger.Debugf("exporter interval value: %s", duration)
```

Reply:
upsss ... it just escaped, I think we can remove it directly